Training/Block/Account/Dashboard/CreditHold.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\Training\Block\Account\Dashboard;
+
+use Magento\Cms\Api\BlockRepositoryInterface;
+use Magento\Framework\View\Element\Template;
+use Magento\Framework\View\Element\Template\Context;
+use Magento\Customer\Model\Session;
+
+/**
+ * Class CreditHold
+ * @package Codifi\Training\Block\Account\Dashboard
+ */
+class CreditHold extends Template
+{
+    /**
+     * Path to the system option Enabled - Credit Hold.
+     *
+     * @var string
+     */
+    const PATH_OPTION_ENABLE = 'codifi/credit_hold/active';
+
+    /**
+     * Path to the system option Value - Message Credit Hold.
+     *
+     * @var string
+     */
+    const PATH_OPTION_MESSAGE = 'codifi/credit_hold/message';
+
+    /**
+     * @var Session
+     */
+    private $session;
+
+    /**
+     * Block Repository Interface
+     *
+     * @var BlockRepositoryInterface
+     */
+    private $blockRepository;
+
+    /**
+     * CreditHold constructor.
+     * @param Session $session
+     * @param BlockRepositoryInterface $blockRepository
+     * @param Context $context
+     * @param array $data
+     */
+    public function __construct(
+        Session $session,
+        BlockRepositoryInterface $blockRepository,
+        Context $context,
+        array $data = []
+    ) {
+        $this->session = $session;
+        $this->blockRepository = $blockRepository;
+        parent::__construct($context, $data);
+    }
+
+    /**
+     * Get credit_hold attribute value
+     *
+     * @return int
+     */
+    public function getCustomerAttr() : int
+    {
+        $customerData = $this->session->getCustomerData();
+        $customerAttribute = $customerData->getCustomAttribute('credit_hold');
```

Comment:
1. getCustomAttribute() method can return null so check it before call getValue()
2. 'credit_hold' can be replaced with constant from previous task